tutorials/CurrentPlaceDetailsOnMap/gradle.properties

```diff
@@ -19,4 +19,4 @@ org.gradle.jvmargs=-Xmx1536m
 # Replace  the value below with your own Google API key for Android app.
 # To learn how to get a Google Maps Platform API key, visit:
 # https://developers.google.com/maps/documentation/android-sdk/start#step_4_get_a_google_maps_api_key
```

Comment:
This URL might break when we update the getting started docs. When we update docs we should search the org repos for URLs and update as needed.